harness/tests/test_gc_harness.py

```diff
@@ -26,10 +25,13 @@ def config(tmp_path: Path) -> Dict[str, Any]:
 @pytest.fixture(params=[0, 1, 5])  # type: ignore
 def to_delete(request: Any, config: Dict[str, Any]) -> List[Dict[str, Any]]:
```

Comment:
question: what is `request` here and why is it `Any`?

Reply:
I have genuinely no idea.  Would you like me to clean this up as part of this work?

Reply:
It doesn’t seem related to this PR. I was just curious.